src/roman_datamodels/testing/factories.py

```diff
@@ -1067,15 +1084,22 @@ def create_ramp_fit_output(**kwargs):
 
     raw = {
         "meta": create_meta(),
-        "slope": _random_array_float32(seg_shape),
-        "sigslope": _random_array_float32(seg_shape),
-        "yint": _random_array_float32(seg_shape),
-        "sigyint": _random_array_float32(seg_shape),
-        "pedestal": _random_array_float32(seg_shape[1:]),
+
+        "slope": _random_array_float32(seg_shape, units=ru.electron / u.s),
+        "sigslope": _random_array_float32(seg_shape, units=ru.electron / u.s),
+        "yint": _random_array_float32(seg_shape, units=ru.electron),
+        "sigyint": _random_array_float32(seg_shape, units=ru.electron),
+        "pedestal": _random_array_float32(seg_shape[1:], units=ru.electron),
+
         "weights": _random_array_float32(seg_shape),
-        "crmag": _random_array_float32(seg_shape),
-        "var_poisson": _random_array_float32(seg_shape),
-        "var_rnoise": _random_array_float32(seg_shape),
+
+        # Magnitude support does not seem to be implemented in ASDF yet
+        # "crmag": u.Magnitude(_random_array_float32(seg_shape) * u.ct / u.s, dtype=np.float32)
```

Comment:
We decided this wasn't a mag and have it below; comment can be deleted.

Reply:
Removed.